### ISSUE TITLE
A user should be either an org owner or org member not both.

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#638]
Should fix issues like: https://github.com/pod4lib/aggregator/issues/637

**Comment by corylown** [comment]
If changing the underlying behavior of the roles proves to be too difficult or problematic it's also possible to change the logic for displaying users, something like:

```
# All users with the role :owner
owners = org.roles.select { |role| role.name == 'owner'  }.flat_map(&:users)

# All users with the role :member who are not also :owner
members = org.roles.select { |role| role.name == 'member'  }.flat_map(&:users).reject { |user| owners.include?(user) }
```

**Comment by cbeer** [comment]
Rolify suggests adding this kind of association:

```
has_many :users, -> { distinct }, through: :roles, class_name: 'User', source: :users
```
https://github.com/RolifyCommunity/rolify/wiki/Usage#finding-roles-through-associations

🤷‍♂️

**Comment by corylown** [comment]
Closed via #700
